src/Compilers/CSharp/Test/Emit3/FieldKeywordTests.cs

```diff
@@ -326,6 +324,198 @@ class C
                 Diagnostic(ErrorCode.ERR_NameNotInContext, "field").WithArguments("field").WithLocation(3, 28));
         }
 
+        [Theory]
+        [InlineData("class")]
+        [InlineData("struct")]
+        [InlineData("ref struct")]
+        [InlineData("record")]
+        [InlineData("record struct")]
+        public void ImplicitAccessorBody_01(string typeKind)
+        {
+            string source = $$"""
+                {{typeKind}} A
+                {
+                    public static object P1 { get; set { _ = field; } }
```

Comment:
>{ get; set { _ = field; } } [](http://example.com/codeflow?start=44&length=27)

Allowing mix of auto- and explicitly-implemented accessors should be tied to language version.